docs/_articles/Power-Production-Distribution-and-Usage-Overview.md

```diff
@@ -0,0 +1,46 @@
+
+* Natural gas: It’s better than coal but the current rush to extract it uses fracking, a controversial method of pumping water underground at high pressure to break the rock apart and allow the gas to flow to the well.The  USGS has this to say about it: [NPR report on USGS statement on fracking causing earthquakes in Oklahoma.](http://www.npr.org/blogs/thetwo-way/2014/05/05/309888859/usgs-okla-fracking-has-increased-chance-of-damaging-quake) There is lots of natural gas available too and because it is overall cleaner than coal will likely be used for quite some time. Again the carbon dioxide and other greenhouse gases would be very detrimental to the environment. 
+
+* Oil: Has many of the same problems that coal and natural gas have however there is not nearly as much of it left and we are using more and more of it all the time. Because oil is used not just for fuel but for plastics and other products it has a higher demand. This demand is at the root of many of the wars that the USA has been involved in recent decades. Please see: [CNN report on Iraq War ten years later](http://www.cnn.com/2013/03/19/opinion/iraq-war-oil-juhasz/)
+
+* Nuclear: Nuclear would be great if it couldn’t be turned into a bomb or the waste potentially pollute the environment for eons. They also require mines to get the uranium and these are also a environmental hazard. Unfortunately this is an energy source that may be increased to keep up with demand and emissions regulation and keep costs competitive with renewable energy sources. 
+
+* Municipal solid waste:  These are touted by some as a renewable energy source. This is trying to cover up a major problem the USA has on its addiction to material things. There would not be such “waste” burned if materials were used efficiently. These power plants are not renewable, half of what they burn is plastic and tires and anything else that may have been thrown in the garbage. 
+
+### Now for the energy sources with the potential to be renewable:
+
+* Human: Human energy is in the form of physical operations performed and mental operations performed. This energy is derived from our consumption of food and water. Industrial Agriculture is where the majority of people get their food and wells and large reservoirs is the source of their water, both of which have geopolitical, environmental and health issues associated with them. 
+ 
+* Geothermal: Geothermal can only be utilized at favorable locations at the moment due to the cost of drilling and it would not be cost effective to use on a small scale. As drilling technology advances and the costs of fossil fuels goes up this will be used by large industrial applications in locations that didn’t used to be possible. As fracking has shown we are still learning the consequences of drilling into the earth so its use should be done with caution.
+
+* Solar: Solar energy is utilized in five ways; hydroelectric, biomass, solar thermal, wind and photovoltaics.
+
```

Comment:
Explain how these resources are all solar? And also, the image should go here, also explain the image rather than just putting it out there, people don't know what theyre looking at.